tests/reference/asr-structs_05-fa98307.stdout

```diff
@@ -1388,11 +1390,5852 @@
                     main_program
                     [_global_symbols]
                     [(SubroutineCall
-                        7 _lpython_main_program
+                        198 _lpython_main_program
                         ()
                         []
                         ()
                     )]
+                ),
+            numpy:
+                (Module
```

Comment:
In this test, I imported `empty` from `numpy`, hence the reference output increased significantly.